.clang-format

```diff
@@ -15,12 +19,17 @@ AlignAfterOpenBracket: AlwaysBreak
 BinPackArguments: false
 BinPackParameters: false
 
-# When constructor initializers exist in the constructor definition, leave the colon as last thing on the original
-# line instead of putting it on the next line.
-BreakConstructorInitializers: AfterColon
 
 # Disallow single statements after if/else/for/while/do without curly braces.
 InsertBraces: true
 
 # Separate definition blocks, including classes, structs, enums, and functions.
 SeparateDefinitionBlocks: Always
+
+# Line up : and , in ctor init list
+AllowShortFunctionsOnASingleLine: false
+BreakConstructorInitializers: BeforeComma
+ConstructorInitializerAllOnOneLineOrOnePerLine: false
```

Comment:
This reflects a personal preference on my part, if folks don't like it I can back it out.

The other changes in .clang-format I think we want...